ska_conda/ska_builder.py

```diff
@@ -6,7 +6,9 @@
 ska_conda_path = os.path.abspath(os.path.dirname(__file__))
 pkg_defs_path = os.path.join(ska_conda_path, "pkg_defs")
 build_list = os.path.join(ska_conda_path, "build_order.txt")
-no_source_pkgs = ['ska3-flight', 'ska3-core', 'ska3-dev', 'ska3-pinned', 'ska3-template']
+no_source_pkgs = ['ska3-flight', 'ska3-core', 'ska3-dev', 'ska3-pinned', 'ska3-template',
```

Comment:
Source probably rename 'no_source_pkgs' to something that makes it more clear that the packages in this list just don't use the local convention we're using that about: home: has the url for a git repository that we build a custom package from using the git tags.